src/components/navigation/HeaderTitle.js

```diff
@@ -39,12 +39,11 @@ class HeaderTitleComponent extends React.PureComponent<Props> {
   }
 
   renderWalletName = () => {
-    const styles = getStyles(this.props.theme)
     if (this.props.selectedWalletName) {
       return (
         <EdgeText>
           {this.props.selectedWalletName + ': '}
-          <EdgeText style={styles.boldText}>{this.props.selectedWalletCurrencyCode}</EdgeText>
+          <EdgeText isBold>{this.props.selectedWalletCurrencyCode}</EdgeText>
```

Comment:
Why this change? This is pretty non-standard as we normally pass in a style to determine bold text